roles/open_zaak_k8s/templates/ingress.yml.j2

```
@@ -0,0 +1,28 @@
+---
+
+kind: Ingress
+apiVersion: networking.k8s.io/v1beta1
+metadata:
+  labels:
+    app.kubernetes.io/name: openzaak
+    app.kubernetes.io/version: "{{ openzaak_version }}"
+    app.kubernetes.io/component: backend
+    app.kubernetes.io/managed-by: Ansible
+  annotations:
+    traefik.ingress.kubernetes.io/rule-type: PathPrefixStrip
+spec:
+{% if openzaak_use_tls %}
+  tls:
+    - hosts:
+        - "{{ openzaak_domain }}"
+      secretName: "openzaak-tls-secrets"
+{% endif %}
+  rules:
+  - host: "{{ openzaak_domain }}"
+    http:
+      paths:
+      - path: /
+        backend:
+          serviceName: openzaak-nginx
+          servicePort: 8080
```

On non-end-of-life kubernetes versions, this is not a valid spec anymore.

```suggestion
      paths:
      - path: /
        backend:
          service: 
            name: openzaak-nginx
            port:
              number: 8080
```

Something like that (untested)

---

Is oke now, tested with tls and without. Works on openshift at least (also creates openshift route)